lib/src/model/model.dart

```diff
@@ -62,10 +75,18 @@ class ModelRelationship {
 class ModelParser {
   static List<T> parseList<T extends Model>(AppStoreConnectClient client, Map<String, dynamic> envelope) {
     final includedModels = _parseIncludes(client, envelope);
-    final modelData = envelope['data'].cast<Map<String, dynamic>>();
-    final modelList = modelData.map((data) => _parseModel(client, data, includedModels)).toList();
 
-    return modelList.cast<T>();
+    final Iterable<Model> modelList;
+    final dataValue = envelope['data'];
+    if (dataValue is Map) {
+      modelList = dataValue.values.map((value) => _parseModel(client, value, includedModels));
+    } else if (dataValue is List) {
```

Comment:
[Frage] Wann verwenden wir die parseList Methode zum Auspacken einer Map Struktur. Stehen dann in den Keys nur unwichtige Informationen drinne?

Reply:
Vor diesem PR wurde davon ausgegangen, dass `data` als Map Struktur vorliegt. Die Endpunkte die ich angesprochen habe, liefern allerdings immer ein Array. Deswegen hab ich die Weiche eingebaut. Wenn ich mir den main Branch anschaue verstehe ich nicht wieso der Code funktionieren soll. Beispielsweise das Laden der [AppStoreVersionLocalizations](https://developer.apple.com/documentation/appstoreconnectapi/appstoreversionlocalizationsresponse) liefert ebenfalls ein Array. Keine Ahnung ob ich alles richtig verstehe.